book/opmanual_autolab/50_autolab_operation_manual/51_autolab_HW_checks_script.md

```diff
@@ -0,0 +1,117 @@
+# Autobot hardware compliance checks {#autolab-autobot-hw-checks status=ready}
+
+Excerpt: run a compliance test and store the information at the right place
+
+We have put together a step-by-step instruction to ensure that every Autobot that enters the Autolab is ready to go. We keep track of them in a GitHub Repository which will be used to keep track of the change history of every Autobot as well as to provide additional information to the user.
+
+<div class='requirements' markdown="1">
+
+Requires: a assembled [Autobot](#autolab-autobot-specs)
+
+Results: a Autobot which is ready to be used in the Autolab.
+
+</div>
+
+## Checklist
+
+### Step 0
+
+Power on the Duckiebot. Then run the following command on your Computer. It will launch a Docker container on your Duckiebot which creates the suitability test report:
+
+    laptop $ docker -H HOSTNAME.local run -it --network host -v /data:/data -v /sys/firmware/devicetree:/devfs:ro duckietown/duckiebot_hw_checks:daffy
+
+### Step 1
+
+If you mounted the Raspberry Pi on top, check that a pin protector is mounted on the pins of the hut to prevent pins from shorting out.
+If no protector is installed yet, then first check that no pins are bent or are making contact and then add the protector. It should slide into place without applying any force. The stl file to 3D print the pin protector can be found in [](#adding-hw-to-bot).
+
+
+### Step 2
+
+Make sure there is a SD card plugged into the Duckiebot for logging using a USB adapter. As the adapter has a LED that might disturb the performance of the other Duckiebots, you need to put a piece of tape on it. Make sure that you cannot see any more light coming from the LED before proceeding. A USB memory stick is fine as well for storage.
+IMPORTANT: the SD card or USB memory stick must be formatted as `ext4`. You can change this by plugging in the memory stick or SD card in the computer, choose `format` and then choose `ext4`.
+
+### Step 3
+
+Check if the standoff on the Duckiebot is mounted and an April tag matching the Duckiebot name (hostname) is attached. The April tag needs to be aligned with the front of the top plate and centered. On the top plate you will find engraved lines that show you where to position it.  the standoff needs to be mounted in the very front of the bot. A picture can be found in [](#fig:standoffs-mounted-autobot)
+
+### Step 4
+
+Check if a rubber duckie sits on the bot. If not, put one on it now.
+As the rubber duckies are rather bad at balancing and lack hands to hold on to the Duckiebot, use double sided tape to fix it to the Duckiebot.
+
+### Step 5
+
+Make sure the back bumper is properly fastened to the Duckiebot (no wiggling allowed).
+
+### Step 6
+
+Check that no cables can interfere with the wheels of the Duckiebot. If necessary, use zip-ties to fix everything properly.
+
+### Step 7
+
+Check that all cables are properly attached. Especially check that the camera cable is plugged in correctly and locked.
```

Comment:
```suggestion
Check that all cables are properly attached. Especially check that the camera cable is plugged in correctly and locked. When the Autobot is on, a small red light appears on the camera. Note that it can sometimes appear but still the camera is badly connected.
```